_oasis

```diff
@@ -185,7 +185,7 @@ Library cli
                     CLIPager
   InternalModules:  CLIData, CLIPluginsLoaded
   CompiledObject:   byte
-  BuildDepends:     oasis, oasis.base, plugin-loader
+  BuildDepends:     oasis, oasis.base, plugin-loader, bytes
```

Comment:
This is actually problematic, because it means we have to depends on very recent version of OCaml (that provides the bytes library).

1. update the ocaml version requirement in this file
2. check the setup.ml generated will work with OCaml 3.12...